### ISSUE TITLE
OpenFortiGUI wrongly prompt the OTP request when there is another error behind

### ISSUE THREAD

**Issue by groud** [theinvisible/openfortigui#89]
Hello, thanks for this very awesome tool !

I figured out that OpenFortiGUI wrongly pops up the OTP request windows while the error behind might be due to something else. For the same configuration file, the GUI popup appears, while using `openfortivpn -c myconfigfile` prompted this:
```
INFO:   Connected to gateway.
INFO:   Authenticated.
INFO:   Remote gateway has allocated a VPN.
Couldn't open the /dev/ppp device: No such device or address
/usr/bin/pppd: Please load the ppp_generic kernel module.

ERROR:  read: Input/output error
INFO:   Cancelling threads...
ERROR:  pppd: The kernel does not support PPP, for example, the PPP kernel driver is not included or cannot be loaded.
INFO:   Terminated pppd.
INFO:   Closed connection to gateway.
INFO:   Logged out.
```

The OTP window is kind of misleading here, so it could be good if openfortigui could instead display an error message. :)

Thanks !

**Comment by theinvisible** [comment]
Hi, 

thanks for your report. As OpenFortiGui searches for specific keywords in the log output it seems there is some mistake in the keyword detection here. Will look into this.

**Comment by theinvisible** [comment]
Hi,

this problem is solved now up from version 0.8.2
